src/routes/(app)/login/admin/+page.server.ts

```diff
@@ -0,0 +1,42 @@
+import { ADMIN_KEY } from "$env/static/private";
```

Comment:
```suggestion
import { env } from "$env/dynamic/private";
```
Secret env variables should be imported dynamically instead since using static will inject them into the code on build time. Pretty sure I've read that this could be a security issue but also, you don't usually deploy with the .env file but have different variable values in prod, so they need to be imported dynamically.

With this change, any use of `ADMIN_KEY` needs to be replaced by `env.ADMIN_KEY`.